Makefile

```diff
@@ -30,3 +31,18 @@ clean:
 	rm -rf $(PYTHON_ENV_ROOT)
 
 envs: env packaging-env
+
+# testing #####################################################################
+$(PYTHON_TESTING_ENV)/.created: REQUIREMENTS.qa.txt
+	rm -rf $(PYTHON_TESTING_DEV) && \
```

Comment:
It looks to me like `PYTHON_TESTING_DEV` is undefined here.

Reply:
You are right. (This typo must be in different projects by now, as the initial commit goes back to 2021 :grinning: )